src/styles/shared/_skeleton.scss

```diff
@@ -36,7 +36,7 @@ $thumbnail-sizes: (
     bottom: 0;
     left: 0;
     display: block;
-    background-color: color('sky');
+    background-color: var(--p-surface-subdued);
```

Comment:
Current color is `#dfe3e8`.

surface-subdued is `#fafbfb` which seems a bit light.

p-surface-neutral (`#e4e5e7`) is nearer to the original.


Perhaps we should be using an [on-surface color](https://github.com/Shopify/polaris-react/blob/master/documentation/Color%20system.md#onsurface) as this is for the "pseudo text that hovers" over a surface background. Perhaps --p-text-disabled (`#8c9196`) but that might be a bit too dark?